app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -233,9 +233,11 @@ const getAnnualIncome = ({ monthlyIncome, annualIncome }) => {
 
 const setApplicationsDefaults = (application) => {
   const applicationWithDefaults = cloneDeep(application)
-
   applicationWithDefaults.annual_income = getAnnualIncome({monthlyIncome: application.monthly_income, annualIncome: application.annual_income})
-
+  // Logic in Lease Section in order to show 'Select One' placeholder on Preference Used if a selection was never made
+  if (applicationWithDefaults.lease && !applicationWithDefaults.lease.no_preference_used && applicationWithDefaults.lease.preference_used == null) {
```

Comment:
The dropdown for `preferenceUsed` works such as:
If the value of `preference_used` is `null`, the dropdown will show "None".
If there is no `preference_used` key in the lease object, the dropdown will show the placeholder "Select One".

Therefore, I created this logic (@akegan this was based on our discussion) to delete the key, and know for certain that `preference_used` was an untouched field when first loading the page.